pkg/vm/engine/tae/dataio/blockio/reader.go

```diff
@@ -80,7 +81,7 @@ func (r *Reader) LoadBlkColumns(
 		if err != nil {
 			return bat, err
 		}
-		data, err := col.GetData()
+		data, err := col.GetData(nil)
```

Comment:
nil, will this work at all?

Reply:
This pr is to allow objectio to support mpool. As for calling objectio to read and write, it needs to be modified in other pr.